src/script/jvm_instance.cpp

```diff
@@ -57,7 +57,7 @@ bool KotlinInstance::get(const StringName& p_name, Variant& r_ret) const {
         return true;
     }
 
-    if (KtFunction* function {kt_class->get_method(SNAME("_get"))}) {
+    if (KtFunction * function {kt_class->get_method(SNAME("_get"))}) {
```

Comment:
Weird formatting

Reply:
Clang not dealing properly with {} initializer again.